### PR TITLE
Add renovate.json

### PR DESCRIPTION
**Pull request by constanzauanini**

### CODE REVIEW

**Review by karthikbtf — APPROVED** [review]
I think we already have something setup for terraware web in Github, this may give us double upgrade PRs?

File: renovate.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (karthikbtf) [inline renovate.json:1]: see web-components, it has a different structure that @charlie-tf came up with

**Comment by constanzauanini** [comment]
> I think we already have something setup for terraware web in Github, this may give us double upgrade PRs?

I don't think so, renovate doc says to create the renovate.json file to enable it if the first "automatic" pr created by renovate was not merged

**Comment by karthikbtf** [comment]
> > I think we already have something setup for terraware web in Github, this may give us double upgrade PRs?
> 
> I don't think so, renovate doc says to create the renovate.json file to enable it if the first "automatic" pr created by renovate was not merged

We have something other than renovate setup. We used to get upgrade PRs for tw. We can disable that layer maybe.

**Review by karthikbtf — APPROVED** [review]

**Pull request merged**